pkg/volume/util/subpath/subpath_linux.go

```diff
@@ -224,8 +240,13 @@ func doBindSubPath(mounter mount.Interface, subpath Subpath) (hostPath string, e
 	kubeletPid := os.Getpid()
 	mountSource := fmt.Sprintf("/proc/%d/fd/%v", kubeletPid, fd)
 
-	// Do the bind mount
+	mountOptions, err := optsGetter.GetMountOptions(subpath.Path)
```

Comment:
why not use `hostutil.NewHostUtil().FindMountInfo(subpath.Path)` directly? this `optsGetter` is added for ut?

Reply:
o, you need to fake in the test

Reply:
yeees